parts/k8s/kubernetesparams.t

```diff
@@ -1,19 +1,3 @@
-{{if .HasAadProfile}}
-    "aadTenantId": {
```

Comment:
The aadTenantId is not referenced anywhere (probably leftover from a prior implementation), so I've cleaned it up while I'm in this area.

Checked as well that the AKS codebase doesn't depend on this in an unexpected way.